### PR TITLE
Thursday July 9, 2015

### PR DESCRIPTION
**Pull request by cbotwell**
:open_file_folder: Starting my project.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,39 @@
...
-
+<nav>
+    <div class="left"></div>
+    <div class="right"></div>
+</nav>
+ <div class="header">
+     <img src="" alt="">
+     <p></p>
+ </div>
+ <ul class="posts">
```
Comment (rtablada) [inline public/index.html]: Don't use an `ul` and `li` here: figure is enough to set apart the picture (you may want to move your `.photo` class though.

`ul` elements should only have child `li` items and include similar if not the same layout of items.
Here you have footer in your `ul`.

File: public/index.html [hunk 2]
```diff
@@ -7,6 +7,39 @@
...
+             <div class="specs">
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+             </div>
+         </figure>
+     </li>
+     <footer>
```
Comment (rtablada) [inline public/index.html]: The two photos should link out to what would likely be sold advertising because the web is a terrible place

**Comment by cbotwell** [comment]
Mostly done. Going to check on other assignments before attempting hard mode.

File: assets/app.scss [hunk 3]
```diff
@@ -1,14 +1,213 @@
...
+            }
+            @include media($mobile) {
+                font-size: 1em;
+                top: 60%;
+                right: 9%;
+            }
+        }
+    }
+}
+body > figure {
```
Comment (rtablada) [inline assets/app.scss]: Better to add a class to this figure rather than using deeply nested selectors

Comment (cbotwell) [inline assets/app.scss]: Fixed, added .post.

**Comment by cbotwell** [comment]
Added pop-out menu on mobile. :iphone:

**Pull request closed without merging**